### DISCUSSION TITLE
white channel data in effects input

### DISCUSSION THREAD

**Discussion by marcobrianza** [forkineye/ESPixelStick#660] [Ideas]
Hello,
using FW Version 4.0-ci5943922254 Build Date Aug 22 2023 - 20:53:21

I am setting up a device with RGBW leds with Artnet as primary input and effects as secondary
I'm interested in setting all leds white with "solid" effect (using the white channel) when no artnet is received.
I have set "Add White Channel Data" but I don't see an option to set the white channel.
Is is missing or I did not find it?

thank you
Marco

**Comment by MartinMueller2003** [comment]
White channel support is a bit of a misnomer when it comes to the effects. The setting tells the effects to reserve four channels of data for each pixel and to not write any data to the fourth (white) channel.

**Reply by marcobrianza** [reply]
ok, so for what I understand RGBW is supported in a way not to disturb the RGB functionality but not to set the white channel

**Comment by MartinMueller2003** [comment]
For effects, that is correct. For data from other sources, White channel support depends on the type of pixels selected. Four channels of data are sent to the pixel and the data source is expected to provide data in four channel groups.
